test/parallel/test-http-response-status-message.js

```diff
@@ -41,7 +56,7 @@ var server = net.createServer(function(connection) {
 var runTest = function(testCaseIndex) {
   var testCase = testCases[testCaseIndex];
 
-  http.get({ port: common.PORT, path: testCase.path }, function(response) {
+  http.get({port: common.PORT, path: testCase.path}, function(response) {
```

Comment:
@Fishrock123 Fair enough. I've removed the changes that only affect spacing on the opening and closing curly braces. I did leave a few that normalize it when there's only spacing on one side or the other. In other words, if there's a line like this:

    var foo = { a: b};

...it gets changed to...

    var foo = {a: b};

If that still rankles, I can remove those as well. 

Anyway, PTAL!